src/utils/template-utils.ts

```diff
@@ -0,0 +1,24 @@
+import type { AST } from "svelte-eslint-parser"
+import voidElements from "./void-elements"
+
+/**
+ * Returns name of SvelteElement
+ */
+export function getNodeName(node: AST.SvelteElement): string {
+  return "name" in node.name ? node.name.name : node.name.property.name
+}
+
+/**
+ * Returns true if Element is custom component
+ */
+export function isCustomComponent(node: AST.SvelteElement): boolean {
+  return node.kind === "component"
+}
```

Comment:
I don't think this function is needed. We can use `node.kind === "component"` directly. Type guards are also effective.